tests/Predis/Command/Redis/TDigest/TDIGESTQUANTILE_Test.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/*
+ * This file is part of the Predis package.
+ *
+ * (c) 2009-2020 Daniele Alessandri
+ * (c) 2021-2023 Till Krüss
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Predis\Command\Redis\TDigest;
+
+use Predis\Command\Redis\PredisCommandTestCase;
+use Predis\Response\ServerException;
+
+class TDIGESTQUANTILE_Test extends PredisCommandTestCase
+{
+    /**
+     * {@inheritDoc}
+     */
+    protected function getExpectedCommand(): string
+    {
+        return TDIGESTQUANTILE::class;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    protected function getExpectedId(): string
+    {
+        return 'TDIGESTQUANTILE';
+    }
+
+    /**
+     * @group disconnected
+     */
+    public function testParseResponse(): void
+    {
+        $this->assertSame(1, $this->getCommand()->parseResponse(1));
+    }
+
+    /**
+     * @group connected
+     * @return void
+     * @requiresRedisBfVersion >= 2.4.0
+     */
+    public function testquantile(): void
```

Comment:
1. Test name should be camelCased
2. Test name should be more specific to represent what we're actually testing, something like `testReturnsValuesBelowGivenQuantile`